spec/jobs/private_body_decrypt_job_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe PrivateBodyDecryptJob, type: :job do
+  let!(:extra_fields) { create(:awesome_proposal_extra_fields, proposal: create(:extended_proposal)) }
```

Comment:
```suggestion
  let!(:proposal) { create(:awesome_proposal_extra_fields, proposal: create(:extended_proposal)) }
```

Should be renamed proposal, we can also create directly a `proposal` rather than `awesome_proposal_extra_fields`